dashboard/src/components/jobs/parameters/JobParams.vue

```diff
@@ -188,19 +201,28 @@ export default class JobParameters extends Vue {
     this.timeParams = timeParams;
   }
   get jobSpec() {
-    return {
-      system_id: this.systemId,
-      time_parameters: this.timeParams,
-      weather_granularity: this.weather_granularity,
-      irradiance_type: this.irradiance_type,
-      temperature_type: this.temperature_type,
-      ...this.jobTypeParams
-    };
+    if (this.isMonthly) {
```

Comment:
A few additional tests to cover this case and a few of the other easy to test changes would be good